tests/integration/targets/efs/playbooks/roles/efs/tasks/main.yml

```diff
@@ -172,6 +172,49 @@
         that: 
           - efs_result is changed
 
+    # ============================================================
+    - name: Add tags to EFS filesystem
+      efs_tag:
+        state: present
+        resource: "{{ created_efs.efs.file_system_id }}"
+        tags:
+          Name: MyTestEFS
+          Env: IntegrationTests
```

Comment:
Please could you add a few extra tagging examples.  Unfortunately there's a few "fun" gotchas that sometimes occur with fetching/setting tags

https://github.com/ansible-collections/amazon.aws/blob/main/tests/integration/targets/ec2_vol/tasks/tests.yml#L254

Reply:
Thanks for reviews. How about https://github.com/ansible-collections/community.aws/pull/608/commits/1c4e32496e9ae714003c5c521a31addfb36755ca?

Reply:
I had to change it a little bit. Moved it to the end of playbook so it won't interfere with other `efs` test & I had to remove some special characters.

```
failed to satisfy constraint: Member must satisfy regular expression pattern: ^([\p{L}\p{Z}\p{N}_.:/=+\-@]*)$; Value 'hello cruel world ❤️' at 'tags.2.member.value' failed to satisfy constraint: Member must satisfy regular expression pattern: ^([\p{L}\p{Z}\p{N}_.:/=+\-@]*)$; Value 'simple_snake_case ❤️' at 'tags.4.member.value' failed to satisfy constraint: Member must satisfy regular expression pattern: ^([\p{L}\p{Z}\p{N}_.:/=+\-@]*)$; Value
```

https://github.com/ansible-collections/community.aws/pull/608/commits/44efc78826204302fec94314ac43b14701a064fd

Reply:
Also, it looks like module defaults are not applied to the new module.
```
  module_defaults:
    group/aws:
      aws_access_key: '{{ aws_access_key }}'
      aws_secret_key: '{{ aws_secret_key }}'
      security_token: '{{ security_token | default(omit) }}'
      region: '{{ aws_region }}'
```

Is there any place to add a new module that belongs to 'group/aws'?